pkg/fuzzer/job.go

```diff
@@ -437,12 +446,16 @@ type hintsJob struct {
 }
 
 func (job *hintsJob) run(fuzzer *Fuzzer) {
+	attempts := 3
```

Comment:
These executions are comparatively rare, so probably it's not worth it to complicate the logic here.

Reply:
Done.